ipaserver/install/dogtaginstance.py

```diff
@@ -278,6 +278,7 @@ def http_proxy(self):
         template = ipautil.template_file(template_filename, sub_dict)
         with open(paths.HTTPD_IPA_PKI_PROXY_CONF, "w") as fd:
             fd.write(template)
+        os.chmod(paths.HTTPD_IPA_PKI_PROXY_CONF, 0o644)
```

Comment:
nit pick: please use ``os.fchmod(fd.fileno(), 0o644)``

Reply:
Is fchmod somehow better than chmod?

Reply:
``fchmod`` operates on the file descriptor and avoids some potential issues like race conditions. In general it's better when the file is already opened.

It's not critical here, but I'd like to move to ``fchmod`` slowly.

Reply:
Thanks for the hint, I updated the code to use fchmod instead.